pages.ml/common/git.md

```diff
@@ -0,0 +1,29 @@
+# git
+
+> പ്രോഗ്രാമുകളുടെ പല പതിപ്പുകൾ പലയിടങ്ങളിലായി സൂക്ഷിക്കുവാനും നിയന്ത്രിക്കുവാനും ഉള്ള സംവിധാനം
+> `commit`, `add`, `branch`, `checkout`, `push` പോലുള്ള ഉപകമാന്റുകൾക്ക് പ്രത്യേക ഡോക്യൂമെന്റഷൻ ഉണ്ട്. അതിനായി `tldr git {{ഉപകമാൻഡ്}}` ഉപയോഗിക്കുക.
+> കൂടുതൽ വിവരങ്ങൾ: <https://git-scm.com/>.
+
+- നിങ്ങൾ ഉപയോഗിക്കുന്ന ഗിറ്റിന്റെ വേർഷൻ കാണുവാൻ:
+
+`git --version`
+
+- സഹായ നിർദേശങ്ങൾ കാണുവാൻ:
+
+`git --help`
+
+- `clone`, `add`, `push`, `log` പോലുള്ള ഉപകമാൻഡുകളുടെ സഹായ നിർദേശങ്ങൾ കാണുവാൻ:
+
+`git help {{ഉപകമാൻഡ്}}`
+
+- ഗിറ്റ് ഉപകമാന്റുകൾ എക്സിക്യൂട്ട് ചെയ്യുവാൻ:
+
+`git {{ഉപകമാൻഡ്}}`
+
+- ഒരു ഇഷ്‌ടാനുസൃത റിപ്പോസിറ്ററി റൂട്ട് പാത്തിൽ ഒരു git ഉപകമാൻഡ് എക്‌സിക്യൂട്ട് ചെയ്യുവാൻ:
+
+`git -C {{പാത/റിപ്പോസിറ്ററി}} {{ഉപകമാൻഡ്}}`
+
+- കോൺഫിഗറേഷൻ ഉപയോഗിച്ച് ഒരു Git ഉപകമാൻഡ് എക്സിക്യൂട്ട് ചെയ്യുവാൻ:
```

Comment:
```suggestion
- ഒരു കോൺഫിഗറേഷൻ സെറ്റ് ഉപയോഗിച്ച് Git സബ്കമാൻഡ് എക്സിക്യൂട്ട് ചെയ്യാൻ:
```

Reply:
for subcommand should we use ഉപകമാൻഡ് or സബ്കമാൻഡ് ? I have used ഉപകമാൻഡ് everywhere else

Reply:
I think we can go with `സബ്കമാൻഡ്` but I can't find the `subcommand` used already on any pages. So, I guess we can choose anything.